classes/controller/AdminController.php

```diff
@@ -2831,7 +2831,7 @@ public function init()
             $this->context->employee->logout();
         }
         if (isset(Context::getContext()->cookie->last_activity)) {
-            if ($this->context->cookie->last_activity + 900 < time()) {
+            if ($this->context->cookie->last_activity + max((int) Configuration::get('PS_COOKIE_LIFETIME_BO') * 3600, 3600) < time()) {
```

Comment:
```suggestion
            $cookieLifetime = (int) Configuration::get('PS_COOKIE_LIFETIME_BO');
            if ($this->context->cookie->last_activity + (!empty($cookieLifetime) ? $cookieLifetime : static::DEFAULT_COOKIE_LIFETIME) < time()) {
```

and add the const `DEFAULT_COOKIE_LIFETIME` :)

Reply:
where do I add DEFAULT_COOKIE_LIFETIME ? 3600 s is ok for u ?

Reply:
Could be in AdminController, or in Configuration, I've no idea what's the best :sweat_smile:

Reply:
Most constants are in config\defines.inc.php

Reply:
It's done @PierreRambaud with a new constant

Reply:
I'm not in favor of adding global const in defines.inc.php, it's pretty old school 😅 
Futhermore, why using max? What if I want to configure the cookie lifetime with 10 minutes, I need to edit defined.inc.php and data in the database? it's weird 🤔 
I think you must remove the max and use a class constant.